lesson_27/kimberleeh/src/pages/Home/components/ProgramList/ProgramList.tsx

```diff
@@ -0,0 +1,43 @@
+import './ProgramList.scss';
+
+import { Program, ProgramProps } from '../Program/Program.tsx';
+import React, { useEffect, useState } from 'react';
+
+
+export const ProgramList: React.FC = () => {
+    const [programs, setPrograms] = useState<ProgramProps[]>([]);
+    
+    useEffect(() => {
+      const fetchPrograms = async () => { 
+        try {
+          const response = await fetch('http://localhost:5173/programs');
+          if (!response.ok) {
+            throw new Error('HTTP error! Status: ${response.status}');
+          }
+          const data = await response.json();
+          console.log('Fetched programs:', data);
+          setPrograms(data.map((program: any) => ({
+            id: program.id,
+            title: program.name,
```

Comment:
The list of `Program` objects don't have a name property. You should read the API code to understand what you're going to get back.